Compentio.SourceMapper.Tests/MapperGenerators/CodeSourceGeneratorTests.cs

```diff
@@ -0,0 +1,144 @@
+        public void GenerateDependencyInjectionExtensions_DotNetCoreStrategy_EmptyDiagnostics()
+        {
+            // Arrange
+            _mockSourcesMetadata.Setup(s => s.DependencyInjection.DependencyInjectionType).Returns(DependencyInjectionType.DotNetCore);
+
+            // Act
+            _codeSourceGenerator.GenerateDependencyInjectionExtensions(_generatorExecutionContext);
+            var diagnostics = _generatorExecutionContext.Compilation.GetDiagnostics();
+
+            // Assert
+            diagnostics.Should().NotBeNull();
+            diagnostics.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void GenerateDependencyInjectionExtensions_AutofacStrategy_EmptyDiagnostics()
+        {
+            // Arrange
+            _mockSourcesMetadata.Setup(s => s.DependencyInjection.DependencyInjectionType).Returns(DependencyInjectionType.Autofac);
+
+            // Act
+            _codeSourceGenerator.GenerateDependencyInjectionExtensions(_generatorExecutionContext);
+            var diagnostics = _generatorExecutionContext.Compilation.GetDiagnostics();
+
+            // Assert
+            diagnostics.Should().NotBeNull();
+            diagnostics.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void GenerateDependencyInjectionExtensions_StructureMapStrategy_EmptyDiagnostics()
+        {
+            // Arrange
+            _mockSourcesMetadata.Setup(s => s.DependencyInjection.DependencyInjectionType).Returns(DependencyInjectionType.StructureMap);
+
+            // Act
+            _codeSourceGenerator.GenerateDependencyInjectionExtensions(_generatorExecutionContext);
+            var diagnostics = _generatorExecutionContext.Compilation.GetDiagnostics();
+
+            // Assert
+            diagnostics.Should().NotBeNull();
+            diagnostics.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void GenerateMapping_NoMappers_EmptyDiagnostics()
+        {
+            // Arrange
+            _mockSourcesMetadata.Setup(s => s.Mappers).Returns(new List<IMapperMetadata>());
+
+            // Act
+            _codeSourceGenerator.GenerateMappings(_generatorExecutionContext);
+            var diagnostics = _generatorExecutionContext.Compilation.GetDiagnostics();
+
+            // Assert
+            diagnostics.Should().NotBeNull();
+            diagnostics.Should().BeEmpty();
+        }
+
+        [Fact]
+        public void GenerateMapping_MappersWithoutMethods_EmptyDiagnostics()
+        {
+            // Arrange
+            _mockMapperMetadata.Setup(m => m.MethodsMetadata).Returns(new List<IMethodMetadata>());
+
+            // Act
+            _codeSourceGenerator.GenerateMappings(_generatorExecutionContext);
+            var diagnostics = _generatorExecutionContext.Compilation.GetDiagnostics();
+
+            // Assert
+            diagnostics.Should().NotBeNull();
+            diagnostics.Should().BeEmpty();
+        }
+
+        private static string GetTestCode()
```

Comment:
1. Why static? 
2. It can be a property:
 ```cs 
private string TestCode => "....";
```

Reply:
Fixed.